docs/index.md

```diff
@@ -2,24 +2,24 @@
 
 ---
 
-## <img class="dcr-icon" src="/img/dcr-icons/decred-logo.svg" /></i> What is Decred?
+## <img class="dcr-icon" alt="logo" src="/img/dcr-icons/decred-logo.svg" /> What is Decred?
 Decred is a cryptocurrency, similar to Bitcoin, with a strong focus on community input, open governance and sustainable funding and development. It utilizes a hybrid "proof-of-work" and "proof-of-stake" mining system to ensure that a small group cannot dominate the flow of transactions or make changes to Decred without the input of the community. A unit of currency is called a 'decred' (DCR). To ensure the integrity of the currency and prevent people from making fraudulent transactions or creating their own coins, Decred uses a method of recording transactions known as a blockchain.
 
 ---
 
-<more information to link to Decred governance, tech and dev pages>
```

Comment:
To be honest I think we should just remove this comment and the comment below - it doesn't seem like whoever wrote them is going to come back and implement them.